_rules/SC1-1-1-aria-describedby.md

```diff
@@ -104,6 +106,14 @@ The resulting assertion is as follows,
 | outcome     | Failed
```

Comment:
Line 72: "If no, could you suggest a...."

Reply:
Thanks